internal/gengapic/gengapic.go

```diff
@@ -652,6 +666,33 @@ func buildAccessor(field string) string {
 	return ax.String()
 }
 
+func (g *generator) lookupFieldType(msgName, field string) (typ descriptor.FieldDescriptorProto_Type) {
```

Comment:
Consider not using naked returns for this function. See https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters, in particular, "Naked returns are okay if the function is a handful of lines. Once it's a medium sized function, be explicit with your return values."

Reply:
Done